bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/icon/FontAwesome5CssReference.java

```diff
@@ -0,0 +1,32 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.icon;
+
+import de.agilecoders.wicket.webjars.request.resource.WebjarsCssResourceReference;
+
+/**
+ * reference for font awesome 5.x css
+ */
+public class FontAwesome5CssReference extends WebjarsCssResourceReference {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * Singleton instance of this reference
+     */
+    private static final class Holder {
+
+        private static final FontAwesome5CssReference INSTANCE = new FontAwesome5CssReference();
+    }
+
+    /**
+     * @return the single instance of the resource reference
+     */
+    public static FontAwesome5CssReference instance() {
+        return Holder.INSTANCE;
+    }
+
+    /**
+     * Private constructor.
+     */
+    private FontAwesome5CssReference() {
+        super("font-awesome/current/css/all.css");
```

Comment:
How does this figures out to use v5 and not v4 ?
Is v4 in a different folder ? E.g. `font-awesome4/...` ?

Reply:
the css for v4 is in a file `font-awesome.css` while that for v5 is in `all.css`. But the problem is another one IMHO, I'll write that in a general comment below.